.github/workflows/main.yml

```diff
@@ -6,8 +6,9 @@ jobs:
     strategy:
       matrix:
         ruby: ['2.5.x', '2.6.x']
-        rails: ['rails_5_1', 'rails_5_2', 'rails_6_0']
-        opal: ['opal_1_0', 'opal_master']
+        rails: ['rails_5_1', 'rails_5_2', 'rails_6_0', 'rails_6_1']
+        # opal: ['opal_1_0', 'opal_master']
```

Comment:
Did opal_master give problems with rails 6.1?

Reply:
opal_master tests doen't run with any rails versions IMHO

I did some sanity checks with rails 6.1 and ran the test suite, which produced 1 error (same as `master`) locally